pontoon/teams/templates/teams/includes/projects.html

```diff
@@ -1,22 +1,33 @@
 {% import 'projects/widgets/project_list.html' as ProjectList with context %}
 
+{% set locale_projects = locale.available_projects_list() %}
+{# Hide if team has any projects #}
+{% set hide_table = (locale.project_set.visible().count() == 0) %}
+{# Hide if there are any projects available for request #}
+{% set hide_request_toggle = projects.exclude(slug__in=locale_projects).count() == 0 %}
```

Comment:
Thank you for your feedback! Very informative as usual!

> please move this variable definitions to the view and pass them to templates as context

This will take some time. Perhaps, I'll have it tomorrow. Otherwise only on weekends. I hope it's not a problem.

Reply:
Definitely not a problem. :)